### PR TITLE
Add the inert attribute to list of features

### PR DESCRIPTION
**Pull request by melanierichards**

### CODE REVIEW

**Comment by alrra** [comment]
Thanks, @melanierichards!

File: status.json [hunk 1]
```diff
@@ -5213,5 +5213,17 @@
     "id": 5663606012116992,
     "uservoiceid": 10938981,
     "msdn": "https://developer.mozilla.org/en-US/docs/Web/CSS/display"
+  },
+  {
+    "name": "inert attribute",
+    "category": "Accessibility",
+    "link": "https://github.com/WICG/inert",
+    "summary": "The inert attribute would allow web authors to mark parts of the DOM tree as inert:\r\n\r\nWhen a node is inert, then the user agent must act as if the node was absent for the purposes of targeting user interaction events, may ignore the node for the purposes of text search user interfaces (commonly known as \"find in page\"), and may prevent the user from selecting text in that node.",
```
Comment (alrra) [inline status.json:5221]: FYI: I removed the `\r\n\r\n`.

Comment (melanierichards) [reply status.json:5221]: Cool! I also had a missing space somewhere but I think you might have fixed that before I rebased and pushed the change ;]

Comment (alrra) [reply status.json:5221]: > Cool! I also had a missing space somewhere but I think you might have fixed that before I rebased and pushed the change 

Yes.

**Pull request closed without merging**